btr-web/btr-main-app/cypress/e2e/pages/reviewConfirm.cy.ts

```diff
@@ -4,10 +4,12 @@ import payFeesForBtrRegsigin from '../../fixtures/payFeeForBtrRegsigin.json'
 describe('pages -> Review and Confirm', () => {
   beforeEach(() => {
     cy.visit('/')
+    cy.wait(1000)
```

Comment:
Is it really necessary at so many places ? Also, can it be shortened, this will create huge delay in builds the more tests we have.

Reply:
well it seems to try testing before the page is ready otherwise which is why it fails a lot